python/ray/dashboard/client/src/common/util.ts

```diff
@@ -0,0 +1,20 @@
+export const getWeightedAverage = (
```

Comment:
Consider adding a unit test for this

Reply:
It isn't a new function, I just moved it, but you're right, it would be good to test.